ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/processes/panel/ProcessesPanelView.java

```diff
@@ -154,5 +160,13 @@ void addWidget(
 
     /** Is called when user has double clicked on console tab to maximize/restore the console. */
     void onToggleMaximizeConsole();
+
+    /**
+     * Is called when `Add Tab` button has been clicked.
```

Comment:
the same comment as [here](https://github.com/eclipse/che/pull/6156#discussion_r137267549)